src/js/me-i18n-locale-en.js

```diff
@@ -33,7 +33,7 @@
             'mejs.fullscreen' : 'Fullscreen',
 
             // mep-feature-jumpforward
-            'mejs.time-jump-forward': 'Jump forward %1 seconds',
+            'mejs.time-jump-forward': 'Jump forward {0} {0|plural:second:seconds}',
```

Comment:
Yeah it's something I discussed and I think you are right. I'll use an array to account for this and adjust code; other than that are you Ok with this approach?